src/components/SystemForActionButton.js

```diff
@@ -0,0 +1,133 @@
+import * as reactIcons from '@patternfly/react-icons';
+import * as reactRouterDom from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import PropTypes from 'prop-types';
+import {
+    Button, Modal
+} from '@patternfly/react-core';
+import { getRegistry } from '@redhat-cloud-services/frontend-components-utilities/files/Registry';
+import reducers from '../store/reducers';
+import RemediationDetailsSystemDropdown from './RemediationDetailsSystemDropdown';
+import ConfirmationDialog from './ConfirmationDialog';
+import { deleteRemediationIssueSystem } from '../actions';
+import { getSystemName } from '../Utilities/model';
+
+const SystemForActionButton = ({ issue, remediation, onDelete }) => {
+    const [ deleteDialogOpen, setDeleteDialogOpen ] = useState(false);
+    const [ InventoryTable, setInventoryTable ] = useState();
+    const [ open, setOpen ] = useState(false);
+    const [ system, setSystem ] = useState({});
+    const [ page, setPage ] = useState(1);
+    const [ pageSize, setPageSize ] = useState(50);
+    const inventory = useRef(null);
+
+    // eslint-disable-next-line react/display-name
+    const detailDropdown = (remediation, issue) => (system) => (
+        <RemediationDetailsSystemDropdown remediation={ remediation } issue={ issue } system={ system } />
+    );
+
+    const loadInventory = async () => {
+        const {
+            inventoryConnector,
+            mergeWithEntities,
+            INVENTORY_ACTION_TYPES
+        } = await insights.loadInventory({
+            react: React,
+            reactRouterDom,
+            reactCore,
+            reactIcons,
+            pfReactTable
+        });
+
+        getRegistry().register({
+            ...mergeWithEntities(reducers.inventoryEntitiesReducer({ INVENTORY_ACTION_TYPES, detailDropdown: detailDropdown(remediation, issue) })())
+        });
+
+        const { InventoryTable } = inventoryConnector();
+        setInventoryTable(() => InventoryTable);
+    };
+
+    useEffect(() => {
+        loadInventory();
+    }, []);
+
+    const onRefresh = (options) => {
+        if (inventory && inventory.current) {
+            setPage(options.page);
+            setPageSize(options.per_page);
+            inventory.current.onRefreshData(options);
+        }
+    };
+
+    return (
+        <React.Fragment>
+            <Button
+                variant='link' onClick={ () => setOpen(true) }>
+                { issue.systems.length }
+            </Button>
+            <Modal
+                className="ins-c-dialog"
+                width={ '50%' }
+                title={ `System${issue.systems.length > 1 ? 's' : ''} for action ${issue.description}` }
+                isOpen={ open }
+                onClose={ () => setOpen(false) }
+                isFooterLeftAligned
+            >
+                { InventoryTable && <InventoryTable
+                    ref={ inventory }
+                    items={ issue.systems }
```

Comment:
These should be ordered the same way they were in the old table

Reply:
The system name should appear as a hyperlink (i.e. blue) ideally opening a new tab when clicked

![Screenshot_20200129_081714](https://user-images.githubusercontent.com/167352/73335517-efc9bc00-426f-11ea-859e-48c2559562fb.png)